client/lib/css-hot-reload/index.js

```diff
@@ -0,0 +1,87 @@
+/**
+ * External dependencies
+ */
+var io = require( 'socket.io-client' );
```

Comment:
Any reason to use `require` here but imports below? Does it have to do with the naming of `socket.io-client`?